doc/source/examples/dem/3d-small-scale-rotating-drum/3d-small-scale-rotating-drum.rst

```diff
@@ -0,0 +1,201 @@
+
+
+
+Description of the Case
+-----------------------
+
+This example simulates a rolling regime in a small scale rotating drum. First, we use Lethe-DEM to fill the bed with 20000 particles. We enable check-pointing in order to write the DEM checkpoint files for the packing which will be used as the starting point of the DEM simulation of the rotating drum. The solver ``dem_3d`` is used to simulate the behaviour of dry granular flow within the rotating drum.
+
+
+
+Parameter File
+--------------
+
+Mesh
+~~~~~
+
+In this example, we choose a ``cylinder`` grid type to create a cylinder. Grid arguments are the radius of the cylinder (0.056 m) and half-length (0.051 m), respectively.  The grid is refined 3 times using the ``set initial refinement`` parameters. The ``expand particle-wall contact search`` is used in concave geometries to enable extended particle-wall contact search with boundary faces of neighbor cells for particles located in each boundary cell (for more details see `Rotating Drum example <../rotating-drum/rotating-drum.html>`_).
+
+.. code-block:: text
+
+    subsection mesh
+        set type                 				= dealii
+        set grid type      	     				= cylinder
+        set grid arguments       				= 0.056:0.051
+        set initial refinement   				= 3
+    end
+
+
+Packing Information
+~~~~~~~~~~~~~~~~~~~~
+
+An insertion box is defined inside the cylindrical domain, inserting 8000 particles every 0.5 seconds while the cylinder is at rest. It is important to note the size of the insertion box to make sure it is completely inside our geometry. Otherwise, particles will be lost during the insertion stage.
+
+.. code-block:: text
+
+
+    subsection insertion info
+    	set insertion method				        = non_uniform
+    	set inserted number of particles at each time step      = 8000
+    	set insertion frequency            		 	= 100000
+    	set insertion box minimum x            	 	= -0.05
+    	set insertion box minimum y            	        = 0.
+    	set insertion box minimum z            	        = -0.04
+    	set insertion box maximum x            	        = 0.05
+    	set insertion box maximum y           	 	= 0.04
+    	set insertion box maximum z            	        = 0.04
+    	set insertion distance threshold			= 1.1
+    	set insertion random number range			= 0.05
+    	set insertion random number seed			= 19
+    end
+
+Restart files are written once the packing ends. The restart files are used to start the DEM simulation with the imposed rotating boundary condition.
+
+Lagrangian Physical Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The particles are mono-dispersed with a radius of 0.0015 m and a density of 2500 kg/m3, respectively. All other particles' physical parameters are taken arbitrary and should be changed based on the physical properties and the experimental values obtained through calibration.
```

Comment:
You could say "In this example, we adopt arbitrary values for the physical properties of the particles.". I wouldn't mention the future measurement of the physical properties of the particles for this example.